crates/egui_extras/src/syntax_highlighting.rs

```diff
@@ -37,6 +31,13 @@ pub fn highlight(ctx: &egui::Context, theme: &CodeTheme, code: &str, language: &
     })
 }
 
+// MEMBRANE: moved impl out of highlight fn to suppress warning
+impl egui::util::cache::ComputerMut<(&CodeTheme, &str, &str), LayoutJob> for Highlighter {
+    fn compute(&mut self, (theme, code, lang): (&CodeTheme, &str, &str)) -> LayoutJob {
+        self.highlight(theme, code, lang)
+    }
+}
+
```

Comment:
Hmm interesting, I don't see any warnings for this code

Reply:
You mean before the change you don't see this (screenshot from vscode, but also seeing on the command line)?
<img width="902" alt="Screenshot 2024-09-04 at 6 20 29 PM" src="https://github.com/user-attachments/assets/0bd86ce8-22a1-45d3-9167-d7064f9f2600">